pwndbg/commands/xor.py

```diff
@@ -44,7 +51,14 @@ def memfrob(address, count):
     Run the memfrob command on a region of memory
     '''
     if not isinstance(address, six.integer_types):
-        address = int(address, 16)
+        try:
+            address = int(address, 16)
```

Comment:
Same logic as in `xor` command. Isn't there any way to just forward it?


I have tried some stuff like:
```python
memfrob = functools.partial(xor, key=42)
```
even wrapping it with:
```python
pwndbg.commands.Command(pwndbg.commands.OnlyWhenRunning(...))
```

didn't really help...

And this:
```python
@pwndbg.commands.Command
@pwndbg.commands.OnlyWhenRunning
def memfrob(address, count):
    '''memfrob(address, count)

    Run the memfrob command on a region of memory
    '''
    return xor(address, 42, count)
```

As this happens:
```
pwndbg> memfrob 0x7fffffffd838 10
object of type 'int' has no len()
'xor': xor(address, key, count)

    XOR ``count`` bytes at ``address`` with the key ``key``.
```

Btw. it might be good to have a `debug` flag and if it is turned on, such things would explode instead of just printing out an exception.